### ISSUE TITLE
[Bug]: only default background works for xfce4

### ISSUE THREAD

**Issue by 813ethan** [termux/termux-x11#660]
### Problem description

when using `xfce4-session`, only default background works while other including the ones bundled by the xfce4 package failed to apply (other than xfce-shapes which is the default)

this showed up when running `termux-x11`:
```
** (xfdesktop:6288): WARNING **: 11:27:14.300: Invalid image style, setting to XFCE_BACKDROP_IMAGE_ZOOMED
** (xfdesktop:6288): WARNING **: 11:27:14.427: xfce_backdrop_generate_canvas: Invalid color style
```

### What steps will reproduce the bug?

use termux-x11 app with commit https://github.com/termux/termux-x11/tree/18eab1ce066e7f0e815e09d5bdbdb8d93448f2d3 or later
start `xfce4-session` in any way
change background to any custom ones via xfce4 Settings->Desktop

### What is the expected behavior?

background of xfce4 can be changed

**Comment by twaik** [comment]
Can not reproduce this. Post `xrandr` output.

**Comment by 813ethan** [comment]
Screen 0: minimum 1 x 1, current 1080 x 1473, maximum 32767 x 32767
內置畫面 connected 1080x1473+0+0 0mm x 0mm
   1080x1473     89.96*

**Comment by 813ethan** [comment]
output of `xrandr` before commit 18eab1
Screen 0: minimum 1 x 1, current 1080 x 1473, maximum 32767 x 32767
screen connected 1080x1473+0+0 0mm x 0mm
   1080x1473     89.96*

**Comment by 813ethan** [comment]
working now somehow

**Comment by 813ethan** [comment]
working only if CJK workaround is applied

**Comment by twaik** [comment]
CJK workaround is completely unrelated to XFCE backgrounds.

**Comment by 813ethan** [comment]
still investigating, seems to work randomly

**Comment by twaik** [comment]
Try this version. [app-universal-debug.zip](https://github.com/user-attachments/files/16049210/app-universal-debug.zip)

**Comment by twaik** [comment]
@813ethan ?

**Comment by 813ethan** [comment]
seems fine

**Comment by twaik** [comment]
Fixed by 0cfbeb41d75d30fc13d880e8e8898367ca51d02c.
